spec/models/payment_spec.rb

```diff
@@ -67,6 +67,14 @@
       it { should_have(:statut).equal_to(:demande).after_event(:ask_for_modification) }
       it { should_have(:statut).equal_to(:demande).after_event(:ask_for_instruction) }
     end
+
+    describe "en_cours_d_instruction" do
+      let(:payment) { create :payment, :en_cours_d_instruction }
+
+      it { should_have(:statut).equal_to(:en_cours_d_instruction).after_event(:ask_for_validation) }
+      it { should_have(:statut).equal_to(:en_cours_d_instruction).after_event(:ask_for_modification) }
+      it { should_have(:statut).equal_to(:en_cours_d_instruction).after_event(:ask_for_instruction) }
+    end
```

Comment:
Il manque pour tous les statuts l'assertion : 
`it { should_have(:statut).equal_to(?).after_event(:send_in_opal) }` où `?` est le statut attendu

Et pour les actions il manque l'assertion : 
`it { should_have(:action).equal_to(?).after_event(:send_in_opal) }` où `?` est l'action attendue